### PR TITLE
[OGUI-373] Add testing to SQLDataSource & Fix bug of rows returned

### PR DESCRIPTION
**Pull request by graduta**
PR containing: 
* Bug fix in which rows from db were returned from top rather than bottom
* Testing and refactoring for the SQLDataSource

### CODE REVIEW

File: InfoLogger/test/lib/mocha-sqldatasource.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+const SQLDataSource = require('../../lib/SQLDataSource.js');
+const {MySQL} = require('@aliceo2/web-ui');
+
+
+describe('SQLDataSource', () => {
+  const config = {
+    mysql: {
+      host: 'localhost',
+      user: 'root',
+      password: 'admintest',
```
Comment (awegrzyn) [inline InfoLogger/test/lib/mocha-sqldatasource.js]: We can't have any password in the code, even dummy ones.

File: InfoLogger/test/lib/mocha-sqldatasource.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+      database: 'INFOLOGGER',
+      port: 8909
```
Comment (awegrzyn) [inline InfoLogger/test/lib/mocha-sqldatasource.js]: I have DB running at different port, maybe let's just have MySQL checks only at CI? This is already the case for https://github.com/AliceO2Group/WebUi/blob/dev/Framework/Backend/test/mocha-db.js

Comment (graduta) [reply InfoLogger/test/lib/mocha-sqldatasource.js]: These are just unit tests and will not make use of any mysql instances. The configuration is only used to mock creating an instance of mysql driver

File: InfoLogger/test/lib/mocha-sqldatasource.js (hunk 1 above)
Comment (graduta) [reply InfoLogger/test/lib/mocha-sqldatasource.js]: them

File: InfoLogger/test/lib/mocha-sqldatasource.js (hunk 2 above)
Comment (awegrzyn) [reply InfoLogger/test/lib/mocha-sqldatasource.js]: Ah okay, maybe just use values from `config-default.js`?

Comment (graduta) [reply InfoLogger/test/lib/mocha-sqldatasource.js]: Updated to use `config-default.js`

**Pull request merged**